foundryContract/src/BatchSubmission.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.20;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./interfaces/IAggregatorSubmit.sol";
+
+contract BatchSubmission is Ownable {
+    uint256 maxSubmission = 50;
+    address[] public oracleAddresses;
+    mapping(address => bool) oracles;
+
+    error OnlyOracle();
+    error InvalidOracle();
+    error InvalidSubmissionLength();
+
+    event OracleAdded(address oracle);
+    event OracleRemoved(address oracle);
+    event MaxSubmissionSet(uint256 maxSubmission);
+
+    constructor() Ownable(msg.sender) {}
+
+    function getOracles() public view returns (address[] memory) {
+        return oracleAddresses;
+    }
+
+    function addOracle(address _oracle) public onlyOwner {
+        oracleAddresses.push(_oracle);
+        oracles[_oracle] = true;
+        emit OracleAdded(_oracle);
+    }
+
+    function removeOracle(address _oracle) public onlyOwner {
+        bool isOracleRemoved = false;
+        for (uint256 i = 0; i < oracleAddresses.length; ++i) {
+            if (oracleAddresses[i] == _oracle) {
+                address last = oracleAddresses[oracleAddresses.length - 1];
+                oracleAddresses[i] = last;
+                oracleAddresses.pop();
+                isOracleRemoved = true;
+                delete oracles[_oracle];
+                break;
+            }
+        }
+        if (isOracleRemoved) emit OracleRemoved(_oracle);
+        else revert InvalidOracle();
+    }
+
+    function setMaxSubmission(uint256 _maxSubmission) public onlyOwner {
+        maxSubmission = _maxSubmission;
+    }
+
+    modifier onlyOracle() {
+        if (!oracles[msg.sender]) revert OnlyOracle();
+        _;
+    }
+
+    function batchSubmit(
+        address[] memory _aggregators,
+        uint256[] memory _roundIds,
+        int256[] memory _submissions
+    ) public onlyOracle {
+        if (
+            !(_aggregators.length == _roundIds.length &&
+                _aggregators.length == _submissions.length)
+        ) revert InvalidSubmissionLength();
+        for (uint i = 0; i < _aggregators.length; i++) {
```

Comment:
In couple functions above, we use explicit `uint256`. Let's be explicit here as well.

```suggestion
        for (uint256 i = 0; i < _aggregators.length; i++) {
```